File: src/main/java/com/mallang/auth/exception/IncorrectUseAuthAtException.java
package com.mallang.auth.exception;

import static org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR;

import com.mallang.common.execption.ErrorCode;
import com.mallang.common.execption.MallangLogException;

public class IncorrectUseAuthAtException extends MallangLogException {

    public IncorrectUseAuthAtException() {
        super(new ErrorCode(INTERNAL_SERVER_ERROR, "@Auth 어노테이션을 잘못 사용했습니다."));
    }
}

File: src/main/kotlin/com/mallang/auth/exception/IncorrectUseAuthAtException.kt
package com.mallang.auth.exception

import com.mallang.common.execption.ErrorCode
import com.mallang.common.execption.MallangLogException
import org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR

class IncorrectUseAuthAtException : MallangLogException(
        ErrorCode(INTERNAL_SERVER_ERROR, "@Auth 어노테이션을 잘못 사용했습니다.")
)
